dndwiki/classes/class-gunslinger.xml

```diff
@@ -0,0 +1,874 @@
+    </sheet>
+    <rules />
+  </element>
+  <element name="Holy Aura" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_HOLY_AURA">
+    <description>
+      <p>Once you reach 15th level, you become permanently protected by a protection from evil and good spell which cannot be suppressed or dispelled.</p>
+    </description>
+    <sheet>
+      <description>You become permanently protected by a protection from evil and good spell which cannot be suppressed or dispelled.</description>
+    </sheet>
+  </element>
+  <element name="Shot of Disruption" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_SHOT_OF_DISRUPTION">
+    <description>
+      <p>Beginning at 18th level, your firearm attacks are devastatingly powerful against fiends and undead, sundering the essence of wicked beings with every shot. If a fiend or undead has 50 hit points or fewer after taking damage from one of your firearm attacks, it must succeed on a Wisdom saving throw (DC = 8 + your Charisma modiier + your proficiency bonus) or be destroyed. On a successful save, the creature becomes frightened of you until the end of your next turn.</p>
+    </description>
+    <sheet>
+      <description>Fiends or undead under 50 HP that are hit by your firearm attacks have to make a wisdom save (DC {{trick_shot:dc}}) or be killed immediately, on a sucess the creatured if frightened until the end of your next turn.</description>
+    </sheet>
+  </element>
+  <element name="Pale Rider" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_PALE_RIDER">
+    <description>
+      <p>Starting at 20th level, death simply can't stop your divine mission. Should you die, you will come back to life 1d4 hours later as if by a true resurrection spell. Once you have used this feature, you can't use it again until 7 days have passed. Additionally, you no longer suffer the drawbacks of old age.</p>
+    </description>
+    <sheet>
+      <description>Once ever 7 days if you die, you are resurrected by a true ressurrection spell after 1d4 hours, you also suffer no drawbacks from old age.</description>
+    </sheet>
+    <rules />
+  </element>
+
+
+  <!-- Firearm Expertise -->
+
+  <element name="Weapon Proficiency (Firearms)" type="Proficiency" source="Critical Role" id="ID_PROFICIENCY_WEAPON_PROFICIENCY_FIREARMS">
+		<requirements>ID_WIKI_CLASS_GUNSLINGER</requirements>
+    <sheet alt="Firearms" />
+		<rules>
+			<grant type="Proficiency" id="ID_CRIT_PROFICIENCY_WEAPON_PISTOL" />
+			<grant type="Proficiency" id="ID_CRIT_PROFICIENCY_WEAPON_MUSKET" />
+			<grant type="Proficiency" id="ID_CRIT_PROFICIENCY_WEAPON_PEPPERBOX" />
+			<grant type="Proficiency" id="ID_CRIT_PROFICIENCY_WEAPON_SCATTERGUN" />
+			<grant type="Proficiency" id="ID_CRIT_PROFICIENCY_WEAPON_BAD_NEWS" />
+			<grant type="Proficiency" id="ID_CRIT_PROFICIENCY_WEAPON_HAND_MORTAR" />
```

Comment:
Maybe not the best idea to insert this in to the Proficiency, for those who doesn't have third-party.index, since i'm pretty sure this will create multiple errors when selecting the Class. (to be honest, not completely sure)

Reply:
Talked to swd about it. He says to just assume core and not worry about the rest.